OrbitCore/SymbolHelper.cpp

```diff
@@ -137,3 +161,36 @@ ErrorMessageOr<ModuleSymbols> SymbolHelper::LoadUsingSymbolsPathFile(
     const std::string& module_path, const std::string& build_id) const {
   return FindSymbols(module_path, build_id, symbols_file_directories_);
 }
+
+ErrorMessageOr<std::string> SymbolHelper::FindDebugSymbolsFile(
+    const std::string& module_path, const std::string& build_id) const {
+  return FindSymbolsFile(module_path, collector_symbol_directories_, build_id);
```

Comment:
Here only the `collector_symbol_directories_` are looked at. This means if the symbols are included in the modules elf file, they are not found. 
Symbols included in the library is a common scenario.

Reply:
Good catch. Will fix

Reply:
done.